hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationManager.java

```diff
@@ -260,6 +262,31 @@ MigrationInfo getActiveMigration() {
         return activeMigrationInfo;
     }
 
+    /**
+     * Acquires promotion commit permit which is needed while running promotion commit
+     * to prevent concurrent commits.
+     * <p>
+     * Normally, promotions are submitted & executed serially
```

Comment:
Hmm, GitHub always marks an `&` in JavaDoc red, do we need to use `&amp;` instead? Didn't find anything about this, just for `@`, `<` and `>`.

Reply:
IDEA doesn't warn for this and renders correctly but haven't tried generating javadoc myself.